api/pom.xml

```diff
@@ -36,6 +36,11 @@
 
   <dependencies>
     <dependency>
+      <groupId>org.hawkular.accounts</groupId>
+      <artifactId>hawkular-accounts-jaxb-adapters</artifactId>
+      <version>1.0.6-SNAPSHOT</version>
```

Comment:
Perhaps, but considering that Maven replaces those automatically during the release, there's no "big" reason to use the var instead of the actual value.